src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,32 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import core.basesyntax.figures.Drawable;
+import core.basesyntax.utils.FigureSupplier;
+
 public class HelloWorld {
+    private static final int DRAWABLE_COUNT = 10;
+
+    public static void main(String[] args) {
+        Drawable[] drawables = generateDrawables();
+
+        for (Drawable drawable : drawables) {
+            drawable.draw();
+        }
+    }
+
+    private static Drawable[] generateDrawables() {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Drawable[] drawables = new Drawable[DRAWABLE_COUNT];
```

Comment:
```suggestion
        Drawable[] drawables = new Drawable[DRAWABLE_COUNT];
```

use Figure type here